README.rst

```diff
@@ -88,19 +88,19 @@ You can get the value of the variables using value(). ex::
 
 Exported Classes:
 
-* `LpProblem` -- Container class for a Linear programming problem
-* `LpVariable` -- Variables that are added to constraints in the LP
-* `LpConstraint` -- A constraint of the general form
+* ``LpProblem`` -- Container class for a Linear programming problem
+* ``LpVariable`` -- Variables that are added to constraints in the LP
+* ``LpConstraint`` -- A constraint of the general form
 
       a1x1+a2x2 ...anxn (<=, =, >=) b
 
-*  `LpConstraintVar` -- Used to construct a column of the model in column-wise modelling
+*  ``LpConstraintVar`` -- Used to construct a column of the model in column-wise modelling
 
 Exported Functions:
 
-* `value()` -- Finds the value of a variable or expression
-* `lpSum()` -- given a list of the form [a1*x1, a2x2, ..., anxn] will construct a linear expression to be used as a constraint or variable
-* `lpDot()` --given two lists of the form [a1, a2, ..., an] and [ x1, x2, ..., xn] will construct a linear epression to be used as a constraint or variable
+* ``value()`` -- Finds the value of a variable or expression
+* ``lpSum()`` -- given a list of the form [a1*x1, a2x2, ..., anxn] will construct a linear expression to be used as a constraint or variable
+* ``lpDot()`` --given two lists of the form [a1, a2, ..., an] and [ x1, x2, ..., xn] will construct a linear epression to be used as a constraint or variable
```

Comment:
> ``lpDot()`` --given two lists of the form [a1, a2, ..., an] and [ x1, x2, ..., xn] will construct a linear epression to be used as a constraint or variable

Is this a typo? "linear epression" should be "linear expression" right?

Reply:
Yep. Feel free to do a PR and I'll merge it.

Reply:
Okay, thanks